topics/contributing/tutorials/create-new-tutorial-content/tutorial.md

```diff
@@ -574,6 +574,51 @@ Rendered:
 > Something really bad can happen here!
 {: .warning}
 
+
+## Tables and Matrices
```

Comment:
I would put this section in the "Content" section. The "Improving the learning experience" is more dedicated to the boxes

Reply:
btw, thanks for adding that